Resources/Prototypes/Entities/Objects/Specific/Xenoarchaeology/item_xenoartifacts.yml

```diff
@@ -54,54 +51,70 @@
       enum.IntercomUiKey.Key:
         type: IntercomBoundUserInterface
   - type: Appearance
+  - type: GuideHelp
+    guides:
+    - Xenoarchaeology
+  # gameplay interactions
+  - type: XenoArtifact
+  - type: Damageable
+  - type: Actions
   - type: Item
     size: Normal
     sprite: Objects/Specific/Xenoarchaeology/item_artifacts.rsi
     heldPrefix: ano01
-  - type: Actions
   - type: Construction
     graph: Artifact
     node: done
-  - type: GuideHelp
-    guides:
-    - Xenoarchaeology
+  - type: ContainerContainer
+    containers:
+      node-container: !type:Container
+        showEnts: False
+        occludes: True
+        ents: []
+  # These components are needed for certain triggers to work.
+  - type: RadiationReceiver
+  - type: Reactive
 
 - type: entity
   parent: BaseXenoArtifactItem
   id: SimpleXenoArtifactItem
   suffix: Simple
   components:
-  - type: Artifact
-    nodesMin: 2
-    nodesMax: 5
+  - type: XenoArtifact
+    nodeCount:
+      min: 2
+      max: 5
```

Comment:
I saw that u were going to do that, but i honestly am not fan of that idea:
1 - It can break maps of downstreams, i think. And for no clearly good reason (see p2)
2 - small-mid artifacts are still quite enjoyable. Small ride, fast and smooth. When value-based matching of trigger-to-effect system will be in place, it could be very nice to have small and simple tool (sometimes).

Also i have tested it A LOT (while bugfixing, sadly) and it felt like almost any graph structure it generates are enjoyable as puzzle. single graph, small 'lines', it all can work and brain enjoys both simple trivial puzzle and complex deep ones. I guess more playtesting will give definitive answer what should be done!